changelog/_template.rst.jinja

```diff
@@ -0,0 +1,33 @@
+.. _vp{{ versiondata.version.replace(".", "p") }}:
+
+{{ top_line }}
+{{ top_underline * top_line|length }}
+{% for section, _ in sections.items() %}
+{% set underline = underlines[0] %}
+
+{% if sections[section] %}
+{% for category, category_data in definitions.items() if category in sections[section] %}
+{{ category_data['name'] }}
+{{ underline * category_data['name']|length }}
+{% if category_data['showcontent'] %}
+{% for text, issues in sections[section][category].items() %}
```

Comment:
Can we somehow sort the `|commands|` and `|tasks|` changes to appear last? That's how it worked before for the manually created changelogs, though replicating this in a jinja template might be a pain

Reply:
Not sure, but if you're able to do it we should should try to do this. 
Essentially if a text starts with `|` then we should try to sort it. This would assume that all |`commands|` and `|tasks|` changes are either not part of other changes or are separate files than other changes.

Reply:
Apparently this isn't possible with jinja templates that can't have custom injections, which shouldn't be a blocker but it could be suggested to the towncrier repo to add config to have a file that can be injected into the template.